### PR TITLE
OracleType enum

### PR DESCRIPTION
**Pull request by kakysha**
It is used in vaults now after refactor

### CODE REVIEW

File: packages/injective-cosmwasm/src/derivative_market.rs [hunk 1]
```diff
@@ -48,8 +49,7 @@ pub struct DerivativeMarket {
     pub ticker: String,
     pub oracle_base: String,
     pub oracle_quote: String,
-    #[serde(default)]
```
Comment (gorgos) [inline packages/injective-cosmwasm/src/derivative_market.rs:51]: Why did you remove this? I believe it was required because 0 is a valid oracle type, but on the Go chain side, the key is omitted from the json if value is zero.

**Review by gorgos — APPROVED** [review]
LGTM but one comment

File: packages/injective-cosmwasm/src/derivative_market.rs (hunk 1 above)
Comment (kakysha) [reply packages/injective-cosmwasm/src/derivative_market.rs:51]: put back Default attribute and implemented Default trait for the type with ::Unspecified as default value
